README.markdown

```diff
@@ -53,6 +53,105 @@ This will generate `*.ts` source files for the given `*.proto` types.
 
 If you want to package these source files into an npm package to distribute to clients, just run `tsc` on them as usual to generate the `.js`/`.d.ts` files, and deploy the output as a regular npm package.
 
+NestJS
```

Comment:
What do you think about breaking this out as a separate `nestjs.markdown` file, and linking to it from the main readme?

Reply:
I kind of agree. We don't want to alienate non nestjs users, actually maybe they will see it and migration over AH AH.

I don't think its a bad idea to separate it into its own markdown. Leaves us more control to write whatever needs to be said for Nestjs without disturbing non nestjs users.

I live the idea.